todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/taskdetail/domain/effect.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.android.architecture.blueprints.todoapp.taskdetail.domain
+
+import com.example.android.architecture.blueprints.todoapp.data.Task
+
+sealed class TaskDetailEffect {
```

Comment:
Please follow the same pattern as the Add/Edit Task feature. There should be a `domain.kt` that includes:

- Model
- Event 
- Effect

And a `logic.kt` file that contains the update function.